controller/controller_server/config.py

```diff
@@ -40,8 +40,8 @@
 PARAMETERS_COPY_TYPE = "copy_type"
 PARAMETERS_CAPACITY_DELIMITER = "="
 PARAMETERS_CAPABILITIES_DELIMITER = "="
-PARAMETERS_OBJECT_ID_INFO_DELIMITER = ":"
-PARAMETERS_OBJECT_IDS_DELIMITER = ";"
+PARAMETERS_OBJECT_ID_INFO_DELIMITER = "parameters_object_id_info_delimiter"
+PARAMETERS_OBJECT_IDS_DELIMITER = "parameters_object_ids_delimiter"
 PARAMETERS_NODE_ID_DELIMITER = ";"
 PARAMETERS_FC_WWN_DELIMITER = ":"
 PARAMETERS_TOPOLOGY_DELIMITER = "/"
```

Comment:
should these also be in config.yaml?
if so, do we have a ticket for it?

Reply:
which parameters?
do you wanna me to do it here?

Reply:
every parameter that is common to the controller and node.
whatever you prefer

Reply:
done